test/assets/static/data.json

```diff
@@ -0,0 +1,5 @@
+{
+    "id": 1,
+    "name": "Foo Bar",
+    "isBot": true
```

Comment:
I'm not sure what `isBot` means, so how about `flag`?

Reply:
I'm sorry, I didn't notice.